spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -64,6 +64,12 @@ def get_value(self, name):
             "it's not installed alongside Spyder. To fix this problem, please "
             "install it in the same environment that you use to run Spyder."
         )
+        reason_mismatched_numpy = _(
+            "The '<tt>{}</tt>' module is required to open this variable and "
+            "it's not installed alongside Spyder. To fix this problem, please "
+            "upgrade `numpy` in the same environment that you use to run "
+            "Spyder to version 1.26.1 or higher."
```

Comment:
```suggestion
            "To fix this problem, please upgrade `numpy` in the same "
            "environment that you use to run Spyder to version 1.26.1 or "
            "higher."
```

I think we shouldn't show the first sentence because it's not so relevant for users. What's important is how they can solve the problem.